package.json

```diff
@@ -7,15 +7,25 @@
     "build": "tsc",
     "start": "node dist/server.js",
     "dev": "nodemon --exec ts-node src/server.ts",
-    "test": "jest"
+    "test": "jest",
+    "migrate": "sequelize db:migrate",
+    "seed": "sequelize db:seed:all",
+    "undo-migrate": "sequelize db:migrate:undo:all"
```

Comment:
add a script for undoing seed, and that of doing everything all at once for simplicity.